canonical-websites/snapcraft.io
# Release UI redux:  history

Continuing redux migration of ReleasesList component:
- moves state of history panel into redux reducers/actions
- moves filtering of history into a selector
- adds tests for all functions

### QA
- ./run or https://snapcraft-io-canonical-websites-pr-1393.run.demo.haus/
- go to Releases page of any snap
- open Redux dev tools
- click around the table, show history of different channels
- opening/closing of panel should show up as actions in redux dev tools
- all release UI functionality should work as before
